WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -84,6 +84,8 @@ extension PostEditor where Self: UIViewController {
                 }
             }
 
+            self.post.confirmedChangesHash = self.post.changesConfirmedContentHashValue()
```

Comment:
I'm planning to change it to a method on an `AbstractPost` when I add the `dateConfirmed` field, so that the operations will timeout after 48h. So it'll be adressed in a follow-up PR :)

Reply:
Sounds good!